### PR TITLE
PB-8 Design Home View

### PR DESCRIPTION
**Pull request by maddiecoco**
- Add InfoHover icon with description. 
- Add LineChart component to track progress.
- Edit home view.

### CODE REVIEW

File: InfoHover.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//
+
+import SwiftUI
+
+struct InfoButtonView: View {
+    let message: String
+    @State private var showingAlert = false
+
+    var body: some View {
+        Button(action: {
```
Comment (kaitlynoflaherty) [inline InfoHover.swift:15]: should we make this a hover rather than a button?

File: LineChart.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+import Charts
+
+struct LineChart: View {
+    // Sample data
+    let data: [ChartData] = [
```
Comment (kaitlynoflaherty) [inline LineChart.swift]: perhaps we should take data in as a variable? not sure where the var would be originated, but then we could call the component with the data passed in

File: posturebest/Views/HomeView.swift [hunk 3]
```diff
@@ -9,21 +9,73 @@ import Foundation
...
+                    ZStack {
+                        // Background box
+                        RoundedRectangle(cornerRadius: 10)
+                            .fill(Color.gray.opacity(0.1))
+                            .frame(width: 350, height: 300)
+                            .shadow(radius: 5)
+                            .padding()
+                        VStack {
+                            Image("TorsoOutline")
+                                .resizable() // Allows you to resize the image
```
Comment (kaitlynoflaherty) [inline posturebest/Views/HomeView.swift]: delete chat comments LOL

**Review by kaitlynoflaherty — CHANGES_REQUESTED** [review]
see in line comments!

File: LineChart.swift (hunk 2 above)
Comment (maddiecoco) [reply LineChart.swift]: yes that was my plan eventually. I just hardcoded it here since we dont have any data rn

**Review by kaitlynoflaherty — APPROVED** [review]

**Pull request merged**